src/CommandHandler.cpp

```diff
@@ -523,7 +520,7 @@ size_t CommandHandler::getNumberOfFileWriterTasks() const {
 std::unique_ptr<FileWriterTask> &
 CommandHandler::getFileWriterTaskByJobID(std::string JobID) {
   for (auto &Task : FileWriterTasks) {
-    if (Task->job_id() == JobID) {
+    if (Task->jobID() == JobID) {
```

Comment:
I have no strong opinion about this and I am horrible at using the std-library myself, but maybe replace the loop with `std::find_if`?